ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/monitor-telemetry-postcard.md

```diff
@@ -0,0 +1,34 @@
+# monitor-telemetry-postcard
```

Comment:
We should have a documentation section under "monitoring" for this feature.

Reply:
Included monitor-telemetry-postcard under monitoring template.